rollup-boot.config.mjs

```diff
@@ -0,0 +1,69 @@
+import { readFileSync } from 'fs';
+
+import { babel } from '@rollup/plugin-babel';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import json from '@rollup/plugin-json';
+import replace from '@rollup/plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+
+const isProd = process.env.NODE_ENV === 'production';
+const prodPlugins = [];
+if (isProd) {
+  prodPlugins.push(terser());
+}
+
+const { version } = JSON.parse(readFileSync('./package.json').toString());
```

Comment:
Can you help me understand the `JSON.parse` and stringifying happening here of JSON source? Apologies if this is a daft question.

Reply:
This module is parsed as an ES module, so `require(...)` can't be used to load JSON files. Instead we have to read it as a file (which returns a `Buffer`), decode that to a `string` and then parse it. Strictly speaking `JSON.parse(readFileSync(path))` does work as well due to an implicit conversion.